libs/template/presets/fulfillment/experience/templates/header-picking-lists.ts

```diff
@@ -0,0 +1,23 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+import { i18n } from '@spryker-oryx/utilities';
+import { HeaderTemplate } from './header';
+
+export const HeaderPickingListsTemplate: ExperienceComponent = {
+  ...HeaderTemplate,
+  id: 'header-picking-lists',
+  components: [
+    {
+      type: 'oryx-content-text',
+      content: {
+        data: {
+          text: `<h4>${i18n('picking.pick-lists')}</h4>`,
```

Comment:
```suggestion
          text: `<h4>PICK LISTS</h4>`,
```

i18n is not supported, we introduced it lately but that's been a mistaken